docs/GetStarted/getting_started_python.md

```diff
@@ -0,0 +1,61 @@
+# Getting Started with Python
+
+IREE has Python bindings geared towards lower level compiler interop that are
+not intended to be a public API, and integration with Python frontends such as
+TensorFlow.
+
+We do not yet provide a pip package for easy installation, so to use IREE's
+Python bindings you must build from source.
+
+## Prerequisites
+
+You should already have IREE cloned and building on your machine. See the other
+[getting started guides](.) for instructions.
+
+> Note:<br>
+> &nbsp;&nbsp;&nbsp;&nbsp;Support is best with Bazel.
+> For CMake (excluding TensorFlow), set the `IREE_BUILD_PYTHON_BINDINGS` option.
+
+## Python Setup
+
+Install a recent version of [Python 3](https://www.python.org/downloads/) and
+[pip](https://pip.pypa.io/en/stable/installing/), if needed.
+
+Install packages:
+
+```shell
+$ pip install numpy
```

Comment:
Sadly, this often produces bad results.

For canonical documentation, I'd recommend: `python3 -m pip install ...`

Why? In multi-python-version regimes (pretty much everything these days), almost the only thing you can easily inspet/control is which interpreter you are invoking. This form makes it explicit.

Reply:
Oh right, thanks for catching that. I've updated to that format and added `python3 -m pip install --upgrade pip`, which we found necessary for `tf-nightly`.